sdk/resourcemanager/Proto.Client/src/Scenarios/CreateSingleVmExample.cs

```diff
@@ -1,4 +1,4 @@
-using Azure.ResourceManager.Core;
+﻿using Azure.ResourceManager.Core;
```

Comment:
remove this file

Reply:
It is removed?

Reply:
![image](https://user-images.githubusercontent.com/53361486/111703837-76868300-87fb-11eb-8c7f-1bf751e591df.png)

Reply:
thats odd its gone now :-)